modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -409,6 +420,12 @@ export default {
     }
   },
   methods: {
+    async preview() {
+      if (!await this.confirmAndCancel()) {
+        return;
+      }
+      window.location = this.original._url;
```

Comment:
Here we're opening in the same tab. In the manager more menu we open in a new tab. Is that intentional? It seems it should be consistent.

Reply:
I don't think you should get hauled off to a new tab if the thing you're navigating to is the thing you just created/edited.

Whereas I can at least see that in the case of the manage view.

Reply:
i had Reasons but they seem not great now .. gut says everyone opens a new tab? I would also axe the confirm-discarding-changes in that change. Make sense?

Reply:
opens a new tab now

Reply:
@boutell the saveDraftAndPreview button still operates as it did (same tab).. is that what you mean?

Reply:
@abea @boutell I'm backed this out to the original way I had it, I think it does make sense:
- In a manager your browsing lots of things, previewing one does not indicate your done with all of them
- In the editor you're focused on one document, and potentially you've made edits to the fields inside. Opening a new tab splits your attention between a thing your editing and a preview that can't reflect those changes. I think in that case it makes sense to force the editor to make a decision about anything they've done up to that point and then change their entire scene to the in-context preview